story-generator/src/app/app.css

```
@@ -1,759 +1,593 @@
-  background: rgba(255, 255, 255, 0.2);
-  color: white;
-  border: 2px solid rgba(255, 255, 255, 0.5);
-  border-radius: 20px;
-  font-size: 0.9rem;
-  font-weight: 600;
-  cursor: pointer;
-  transition: all 0.3s ease;
-  backdrop-filter: blur(5px);
+  background: radial-gradient(circle at top, rgba(59, 12, 71, 0.3), rgba(12, 12, 28, 0.95));
+  color: #f7f1ff;
+  font-family: 'Inter', system-ui, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)